tests/tests/operation_tests.cpp

```diff
@@ -4118,6 +4118,173 @@ BOOST_FIXTURE_TEST_SUITE(operation_tests, clean_database_fixture)
+            const auto bob_owner_authority = authority(1, generate_private_key("bob_owner").get_public_key(), 1);
+
+            acc_create.fee = ASSET("10.000 GOLOS");
+            acc_create.creator = "alice";
+            acc_create.new_account_name = "bob";
+            acc_create.owner = bob_owner_authority;
+            acc_create.active = authority(1, bob_private_key.get_public_key(), 1);
+            acc_create.posting = authority(1, generate_private_key("bob_posting").get_public_key(), 1);
+            acc_create.memo_key = generate_private_key("bob_memo").get_public_key();
+            acc_create.json_metadata = "";
+
+            signed_transaction tx;
+            BOOST_CHECK_NO_THROW(push_tx_with_ops(tx, alice_private_key, acc_create));
+
+            tx.clear();
+
+            BOOST_TEST_MESSAGE("--- Creating account chewy with alice");
+            acc_create.fee = ASSET("0.000 GOLOS");
+            acc_create.creator = "alice";
+            acc_create.new_account_name = "chewy";
+            acc_create.owner = authority(1, generate_private_key("chewy_owner").get_public_key(), 1);
+            acc_create.active = authority(1, generate_private_key("chewy_active").get_public_key(), 1);
+            acc_create.posting = authority(1, generate_private_key("chewy_posting").get_public_key(), 1);
+            acc_create.memo_key = generate_private_key("chewy_memo").get_public_key();
+            acc_create.json_metadata = "";
+
+            tx.operations.push_back(acc_create);
+
+            BOOST_TEST_MESSAGE("--- Creating account donna with alice");
+            acc_create.fee = ASSET("0.000 GOLOS");
+            acc_create.creator = "alice";
+            acc_create.new_account_name = "donna";
+            acc_create.owner = authority(1, generate_private_key("donna_owner").get_public_key(), 1);
+            acc_create.active = authority(1, generate_private_key("donna_active").get_public_key(), 1);
+            acc_create.posting = authority(1, generate_private_key("donna_posting").get_public_key(), 1);
+            acc_create.memo_key = generate_private_key("donna_memo").get_public_key();
+            acc_create.json_metadata = "";
+
+            tx.operations.push_back(acc_create);
+            tx.sign(alice_private_key, db->get_chain_id());
+
+            BOOST_CHECK_NO_THROW(db->push_transaction(tx, 0));
+
+            BOOST_TEST_MESSAGE("--- Set vesting withdraw routes");
+
+            set_withdraw_vesting_route_operation to_chewy;
+            to_chewy.from_account = "bob";
+            to_chewy.to_account = "chewy";
+            to_chewy.percent = 5000;
+
+            set_withdraw_vesting_route_operation to_donna;
+
+            to_donna.from_account = "bob";
+            to_donna.to_account = "donna";
+            to_donna.percent = 5000;
+
+            tx.clear();
+
+            BOOST_CHECK_NO_THROW(push_tx_with_ops(tx, bob_private_key, to_chewy, to_donna));
+
+            BOOST_TEST_MESSAGE("--- Start vesting withdraw");
+
+
```

Comment:
Excessive new line. Do not write excessive new lines. If you want separate code block from another - use comments or BOOST_TEST_MESSAGE.